src/attached_probe.cpp

```diff
@@ -29,226 +28,195 @@ const int BPF_LOG_SIZE = 400 * 1024;
  * `notrace`. However, the ones below are not excluded.
  */
 const std::set<std::string> banned_kretprobes = {
-  "_raw_spin_lock", "_raw_spin_lock_irqsave", "_raw_spin_unlock_irqrestore",
-  "queued_spin_lock_slowpath",
+    "_raw_spin_lock",
+    "_raw_spin_lock_irqsave",
+    "_raw_spin_unlock_irqrestore",
+    "queued_spin_lock_slowpath",
 };
 
-
-bpf_probe_attach_type attachtype(ProbeType t)
-{
-  switch (t)
-  {
-    case ProbeType::kprobe:    return BPF_PROBE_ENTRY;  break;
-    case ProbeType::kretprobe: return BPF_PROBE_RETURN; break;
-    case ProbeType::uprobe:    return BPF_PROBE_ENTRY;  break;
-    case ProbeType::uretprobe: return BPF_PROBE_RETURN; break;
-    case ProbeType::usdt:      return BPF_PROBE_ENTRY;  break;
-    default:
-      std::cerr << "invalid probe attachtype \"" << probetypeName(t) << "\"" << std::endl;
-      abort();
+bpf_probe_attach_type attachtype(ProbeType t) {
+  switch (t) {
+  case ProbeType::kprobe: return BPF_PROBE_ENTRY; break;
+  case ProbeType::kretprobe: return BPF_PROBE_RETURN; break;
+  case ProbeType::uprobe: return BPF_PROBE_ENTRY; break;
+  case ProbeType::uretprobe: return BPF_PROBE_RETURN; break;
+  case ProbeType::usdt: return BPF_PROBE_ENTRY; break;
+  default:
+    std::cerr << "invalid probe attachtype \"" << probetypeName(t) << "\""
+              << std::endl;
+    abort();
   }
 }
 
-bpf_prog_type progtype(ProbeType t)
-{
-  switch (t)
-  {
-    case ProbeType::kprobe:     return BPF_PROG_TYPE_KPROBE; break;
-    case ProbeType::kretprobe:  return BPF_PROG_TYPE_KPROBE; break;
-    case ProbeType::uprobe:     return BPF_PROG_TYPE_KPROBE; break;
-    case ProbeType::uretprobe:  return BPF_PROG_TYPE_KPROBE; break;
-    case ProbeType::usdt:       return BPF_PROG_TYPE_KPROBE; break;
-    case ProbeType::tracepoint: return BPF_PROG_TYPE_TRACEPOINT; break;
-    case ProbeType::profile:      return BPF_PROG_TYPE_PERF_EVENT; break;
-    case ProbeType::interval:      return BPF_PROG_TYPE_PERF_EVENT; break;
-    case ProbeType::software:   return BPF_PROG_TYPE_PERF_EVENT; break;
-    case ProbeType::hardware:   return BPF_PROG_TYPE_PERF_EVENT; break;
-    default:
-      std::cerr << "program type not found" << std::endl;
-      abort();
+bpf_prog_type progtype(ProbeType t) {
+  switch (t) {
+  case ProbeType::kprobe: return BPF_PROG_TYPE_KPROBE; break;
```

Comment:
I prefer the old style with alignment, but can't see any settings in clang format to keep it :(

Reply:
Set `AllowShortCaseLabelsOnASingleLine: false`